<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="1" name="fizzbuzz_tests.fizzBuzzTest-20181018141852" skipped="0" tests="5" time="0.001" timestamp="2018-10-18T14:18:52">
	<testcase classname="fizzbuzz_tests.fizzBuzzTest" name="test_if_10_is_buzz" time="0.000" timestamp="2018-10-18T14:18:52"/>
	<testcase classname="fizzbuzz_tests.fizzBuzzTest" name="test_if_3_is_fizz" time="0.000" timestamp="2018-10-18T14:18:52"/>
	<testcase classname="fizzbuzz_tests.fizzBuzzTest" name="test_if_5_is_buzz" time="0.000" timestamp="2018-10-18T14:18:52"/>
	<testcase classname="fizzbuzz_tests.fizzBuzzTest" name="test_if_9_is_fizz" time="0.000" timestamp="2018-10-18T14:18:52"/>
	<testcase classname="fizzbuzz_tests.fizzBuzzTest" name="test_if_4_is_4" time="0.001" timestamp="2018-10-18T14:18:52">
		<failure message="'fizz' != '4'
- fizz
+ 4
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Users/mikebmc/Documents/job_hunting_2018/applications/Intersection/nr_real_fizzbuzz/src/unittest/python/fizzbuzz_tests.py", line 23, in test_if_4_is_4
    self.assertEqual(fizzbuzz.getFizzBuzz(4), '4')
AssertionError: 'fizz' != '4'
- fizz
+ 4

]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
